### ISSUE TITLE
Featured Product Block having issue with text color - editor side.

### ISSUE THREAD

**Issue by NidhiDhandhukiya74** [woothemes/woocommerce#42386]
### Prerequisites

- [ ] I have carried out troubleshooting steps and I believe I have found a bug.
- [ ] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

When you use Featured Product block and apply any text color.
It is visible on the front side but not able to see any change in editor side.

### Expected behavior

When we choose any color from color palate that choosen color is applied to the editor side as well.

### Actual behavior

When we choose any color from color palate that is not reflected in editor side.

### Steps to reproduce

1. Take Featured Product block.
2. Choose a product.
3. Apply text color from the color palate.
4. Check that in both editor side and user side.

### Video URl for better understanding
URL :- https://share.cleanshot.com/nZGY7mjXHqpB80XPCMVk

### WordPress Environment

Wordpress Environment :- 6.2.1
Theme :- Twenty Nineteen

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by NidhiDhandhukiya74** [comment]
Solution :- 
Css is missing for the editor side as per the chosen color variable.
If we want to resolve this issue we need to add css as per the chosen variable.

For example in the Twenty Nineteen theme primary color is chosen so the below css can resolve the issue.
`.wp-block-woocommerce-featured-product.has-primary-color {
   color: #0073aa;
}`

**Comment by ObliviousHarmony** [comment]
Hello,

This looks like it might be released to our Blocks, so, I'm going to transfer this to the feature plugin's repository.

**Comment by imanish003** [comment]
Hi @NidhiDhandhukiya74,

Thank you for bringing this issue to our attention. I have successfully reproduced the problem using the `Twenty Nineteen` theme.

I have added this issue to our backlog for further investigation and resolution.

cc: @Aljullu It seems that you have also reported a similar issue (#9424). However, it appears that there are additional problems when using the `Twenty Nineteen` theme. Perhaps we should consider adding default styles to all elements within the `Featured Product` block. What do you think?

**Comment by Aljullu** [comment]
> cc: @Aljullu It seems that you have also reported a similar issue (#9424). However, it appears that there are additional problems when using the `Twenty Nineteen` theme. Perhaps we should consider adding default styles to all elements within the `Featured Product` block. What do you think? 

Good catch. They definitely look similar (even though, different). As @NidhiDhandhukiya74, I think we are not adding some specific class or style to the block when the color is edited. I'm not sure I'm following with the "default styles", what do you mean?

**Comment by gigitux** [comment]
Thank you for your report and for contributing to the project!

Given that this is bug is reproducible only on Twenty Nineteen and due to our current priorities, we're closing this issue for now to help manage our backlog. This doesn't mean it won't be revisited in the future—just that our focus is currently elsewhere. If this issue becomes more relevant later on, feel free to reopen it.
